array-methods/map.ts

```diff
@@ -5,3 +5,10 @@ const sale = prices.map((num) => ({
   salePrice: num / 2,
 }));
 console.log(sale);
+
+const format = prices.map((num) => ({
+  price: num.toFixed(2),
+  salePrice: (num / 2).toFixed(2),
+}));
```

Comment:
This one should not be an object. You are just turning the numbers into strings and adding the 2 decimal points with `toFixed`